setup.py

```diff
@@ -9,31 +9,50 @@
 
 version = re.search(
     '^__version__\s*=\s*"(.*)"',
-    open('tecan_od_analyzer/tecan_od_analyzer.py').read(),
-    re.M
-    ).group(1)
+    open("tecan_od_analyzer/tecan_od_analyzer.py").read(),
+    re.M,
+).group(1)
 
 setup(
-    name = "tecan_od_analyzer",
-    version = version,
-    author = "Felix Pacheco Pastor",
-    author_email = "fepac@biosustain.com",
-    description = ("OD data handling for growth curve estimation and visualization of TECAN OD readings"),
-    long_description = long_description, 
+    name="tecan_od_analyzer",
+    version=version,
+    author="Felix Pacheco Pastor",
```

Comment:
Please update the author and email with your information when you get a chance.

Reply:
Ok also changed it in the __init__.py file and updated the version number there to the current version